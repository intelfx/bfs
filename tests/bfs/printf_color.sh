bfs_diff -color -exclude \( -depth 1 -not -name rainbow \) -printf '%H %h %f %p %P %l\n'
