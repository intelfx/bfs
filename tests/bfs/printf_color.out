[01;34m.[0m [01;34m$'./rainbow/\e[1m'[0m $'\e[0m' [01;34m$'./rainbow/\e[1m/'[0m$'\e[0m' [01;34m$'rainbow/\e[1m/'[0m$'\e[0m' 
[01;34m.[0m [01;34m.[0m [01;34m.[0m [01;34m.[0m  
[01;34m.[0m [01;34m.[0m [01;34mrainbow[0m [01;34m./rainbow[0m [01;34mrainbow[0m 
[01;34m.[0m [01;34m./rainbow[0m [01;32mexec.sh[0m [01;34m./rainbow/[0m[01;32mexec.sh[0m [01;34mrainbow/[0m[01;32mexec.sh[0m 
[01;34m.[0m [01;34m./rainbow[0m [01;34m$'\e[1m'[0m [01;34m$'./rainbow/\e[1m'[0m [01;34m$'rainbow/\e[1m'[0m 
[01;34m.[0m [01;34m./rainbow[0m [01;35msocket[0m [01;34m./rainbow/[0m[01;35msocket[0m [01;34mrainbow/[0m[01;35msocket[0m 
[01;34m.[0m [01;34m./rainbow[0m [01;36mbroken[0m [01;34m./rainbow/[0m[01;36mbroken[0m [01;34mrainbow/[0m[01;36mbroken[0m nowhere
[01;34m.[0m [01;34m./rainbow[0m [01;36mchardev_link[0m [01;34m./rainbow/[0m[01;36mchardev_link[0m [01;34mrainbow/[0m[01;36mchardev_link[0m [01;34m/dev/[0m[01;33mnull[0m
[01;34m.[0m [01;34m./rainbow[0m [01;36mlink.txt[0m [01;34m./rainbow/[0m[01;36mlink.txt[0m [01;34mrainbow/[0m[01;36mlink.txt[0m file.txt
[01;34m.[0m [01;34m./rainbow[0m [30;42msticky_ow[0m [01;34m./rainbow/[0m[30;42msticky_ow[0m [01;34mrainbow/[0m[30;42msticky_ow[0m 
[01;34m.[0m [01;34m./rainbow[0m [30;43msgid[0m [01;34m./rainbow/[0m[30;43msgid[0m [01;34mrainbow/[0m[30;43msgid[0m 
[01;34m.[0m [01;34m./rainbow[0m [33mpipe[0m [01;34m./rainbow/[0m[33mpipe[0m [01;34mrainbow/[0m[33mpipe[0m 
[01;34m.[0m [01;34m./rainbow[0m [34;42mow[0m [01;34m./rainbow/[0m[34;42mow[0m [01;34mrainbow/[0m[34;42mow[0m 
[01;34m.[0m [01;34m./rainbow[0m [37;41msugid[0m [01;34m./rainbow/[0m[37;41msugid[0m [01;34mrainbow/[0m[37;41msugid[0m 
[01;34m.[0m [01;34m./rainbow[0m [37;41msuid[0m [01;34m./rainbow/[0m[37;41msuid[0m [01;34mrainbow/[0m[37;41msuid[0m 
[01;34m.[0m [01;34m./rainbow[0m [37;44msticky[0m [01;34m./rainbow/[0m[37;44msticky[0m [01;34mrainbow/[0m[37;44msticky[0m 
[01;34m.[0m [01;34m./rainbow[0m file.dat [01;34m./rainbow/[0mfile.dat [01;34mrainbow/[0mfile.dat 
[01;34m.[0m [01;34m./rainbow[0m file.txt [01;34m./rainbow/[0mfile.txt [01;34mrainbow/[0mfile.txt 
[01;34m.[0m [01;34m./rainbow[0m lower.gz [01;34m./rainbow/[0mlower.gz [01;34mrainbow/[0mlower.gz 
[01;34m.[0m [01;34m./rainbow[0m lower.tar [01;34m./rainbow/[0mlower.tar [01;34mrainbow/[0mlower.tar 
[01;34m.[0m [01;34m./rainbow[0m lower.tar.gz [01;34m./rainbow/[0mlower.tar.gz [01;34mrainbow/[0mlower.tar.gz 
[01;34m.[0m [01;34m./rainbow[0m lu.tar.GZ [01;34m./rainbow/[0mlu.tar.GZ [01;34mrainbow/[0mlu.tar.GZ 
[01;34m.[0m [01;34m./rainbow[0m mh1 [01;34m./rainbow/[0mmh1 [01;34mrainbow/[0mmh1 
[01;34m.[0m [01;34m./rainbow[0m mh2 [01;34m./rainbow/[0mmh2 [01;34mrainbow/[0mmh2 
[01;34m.[0m [01;34m./rainbow[0m ul.TAR.gz [01;34m./rainbow/[0mul.TAR.gz [01;34mrainbow/[0mul.TAR.gz 
[01;34m.[0m [01;34m./rainbow[0m upper.GZ [01;34m./rainbow/[0mupper.GZ [01;34mrainbow/[0mupper.GZ 
[01;34m.[0m [01;34m./rainbow[0m upper.TAR [01;34m./rainbow/[0mupper.TAR [01;34mrainbow/[0mupper.TAR 
[01;34m.[0m [01;34m./rainbow[0m upper.TAR.GZ [01;34m./rainbow/[0mupper.TAR.GZ [01;34mrainbow/[0mupper.TAR.GZ 
